types/three/examples/jsm/nodes/accessors/CameraNode.d.ts

```diff
@@ -0,0 +1,12 @@
+import Object3DNode, { Object3DNodeScope } from './Object3DNode';
+
+export type CameraNodeScope = Object3DNodeScope | typeof CameraNode.PROJECTION_MATRIX;
+
+export default class CameraNode extends Object3DNode {
+    static PROJECTION_MATRIX: 'projectionMatrix';
+
+    // @ts-expect-error
```

Comment:
This a general problem why I didn't want to be too specific in subclasses. In this case I decided to go this way, because I like the scope to be an union type, and I also hate to make PROJECTION_MATRIX an option for Object3DNode (also `typeof` would introduce circular depenency, I would have to be hardcoded). I concluded that the most user friendly one.